Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -385,8 +385,7 @@ def audit_deps
             problem "Dependency '#{dep.name}' was renamed; use new name '#{dep_f.name}'."
           end
 
-          if self.class.aliases.include?(dep.name) &&
-             (dep_f.core_formula? || !dep_f.versioned_formula?)
```

Comment:
This looks like it should already not be complaining for using versioned formulae aliases? Is that not working as expected?

Reply:
Oh, maybe it's because my tap is weird, and we have our versioned formulae named like `formulaN.rb` instead of `formula@N.rb` and create aliases for `formula@N`. So the current master branch would give this audit error if I depend on `formula@N`.

I'll try to figure out why we didn't switch to `formula@N`. My vague memory is that something else complained that my formulae needed to be keg-only, and I wanted to make them conflict with each other instead of being keg-only.

Reply:
Yeh, that's likely it 😭